build.gradle

```diff
@@ -212,10 +199,18 @@ allprojects {
     }
     implementation  group: 'com.fasterxml.jackson.module', name: 'jackson-module-jaxb-annotations', version: '2.10.3'
     implementation 'net.minidev:json-smart:2.4.7'
-  }
 
-  // Global constraints
-  dependencies {
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-json'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-activemq'
```

Comment:
May need activemq-spring as well.  Integration tests in services module use activemq Spring configuration which in the original SDT required activemq-spring.

Reply:
will add these when required

Reply:
Agreed.